### PR TITLE
Added TensorFlow and Keras support; integrated setuptools

### PR DESCRIPTION
**Pull request by SamuelMarks**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,11 @@
+root = true
```
Comment (fediazgon) [inline .editorconfig]: Can you remove this file or add it to the .gitignore?

File: requirements.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+numpy
```
Comment (fediazgon) [inline requirements.txt]: Can you add `scikit-learn` and uncomment one of those?

File: pbt/__init__.py [hunk 3]
```diff
@@ -1,8 +1,32 @@
...
+# -*- coding: utf-8 -*-
+
 import logging
-from logging import NullHandler
+from logging.config import dictConfig as _dictConfig
+from os import path
+
+import yaml
+
+__author__ = 'Giorgio Ruffa <gioruffa@gmail.com>'
```
Comment (xmooner) [inline pbt/__init__.py]: Thanks for contributing to the project!

I am flattered to be included as an author, but most of the code was written by Fernando.
Consider also adding yourself to the list of contributors in the `__credit__` header field.
```suggestion
__author__ = 'Fernando Diaz <fediazgon@gmail.com>, Giorgio Ruffa <gioruffa@gmail.com>'
__credits__ = 'your name and email'
```

**Comment by SamuelMarks** [comment]
Made the changes

Hey also on the Keras vs tf.keras side, I've been playing around with my own library+CLI, and I'm actually looking at supporting:
- Keras (CNTK and Theano)
- tf.keras (TensorFlow)
- torch (PyTorch)

A project like this could be useful for improving the parameters on any of these 4 backends. So let's not remove Keras and replace with tf.keras 😅

**Pull request merged**